bin/scanpy-run-tsne.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import logging
+import matplotlib
+matplotlib.use('Agg')
+from scanpy_wrapper_utils import ScanpyArgParser, comma_separated_list
+from scanpy_wrapper_utils import read_input_object, write_output_object, save_output_plot
+
+
+def main(args):
+    logging.debug(args)
+    import scanpy.api as sc
+
+    adata = read_input_object(args.input_object_file, args.input_format)
+
+    sc.tl.tsne(adata,
```

Comment:
Could we be more consistent with the script naming with respect to function calls? Maybe scanpy-tsne.py for this one? Maybe you even want to use scanpy-tl-tsne.py. Whatever you choose, could you apply the same to all scripts which wrap a single defined Scanpy function? Thank you!